/*package u4.prac_test2;

public class Demo {
	public static void demo() {
		try {
			statement1;
			statement2;
			statement3;
		}
		catch (Exception ex1) {
			statement5;
		}
		finally {
			statement4;
		}
		
		statement6;
	}
	
	public static void methodA() {
		try {
			int[] a = new int[4];
			a[6] = 19;
			System.out.println("Finished!");			
		}
		catch (ArithmeticException ex) {
			System.out.println("Arithmetic Exception!");
		}
		catch (Exception ex) {
			System.out.println("Exception!");
		}
	}
	
	public static void methodB() {
		try {
			int a = 5 / 0;
			System.out.println("Finished!");						
		}
		catch (RuntimeException ex) {
			System.out.println("Runtime Exception!");
		}
		catch (Exception ex) {
			System.out.println("Exception!");
		}
	}	
}
*/